client/js/main.js

```diff
@@ -44,11 +44,80 @@ function getJBData(e) {
 
 function parseInput(e) {
 	e.preventDefault();
-	var value = e.currentTarget.querySelector('#json-input').value;
-	
-	if(value.length > 0) {
+
+	var classifierText = e.currentTarget.querySelector('#classifier-input').value;
+	var jsonText       = e.currentTarget.querySelector('#json-input'      ).value;
+
+	if(classifierText.length > 0) {
+		console.log(`DEBUG: parsing classifierText`);
+		// expecting classifierText to be:
+		// http://classifier_url
+		// bodyparam=VALUE
+		// bodyparam=VALUE
+		// bodyparam=VALUE
+		// bodyparam=VALUE
+
+		let classifierUrl = null;
+		let bodyParamsStrings = [];
+
+		let lines = classifierText.split(/\n/);
+
+		let mUrl = lines[0].match(/^(https?:.*)/);
+		if( mUrl ) {
+			classifierUrl = mUrl[1];
+			console.log(`DEBUG: parsing classifierUrl=${classifierUrl}`);
+		} else {
+			alert(`no classifier url specified in 1st line`);
+		}
+
+		lines.slice(1).forEach( line => {
```

Comment:
wrap in `Array.from()` for use with `forEach`. Avoid arrow function on the client-side. We're not using any polyfills or compiling the JS to be cross-browser compatible.

Reply:
I've become spoiled by developing primarily in Node, forgetting what JS is actually like. Soz.